vnext/Desktop/WebSocket.h

```diff
@@ -127,15 +149,15 @@ class BaseWebSocket : public IWebSocket {
   /// <param name="options">
   /// Map of HTTP header fields sent by the remote endpoint.
   /// </param>
-  virtual void Handshake(const IWebSocket::Options &options);
+  virtual void Handshake();
 
  public:
 #pragma region IWebSocket
 
   /// <summary>
   /// <see cref="IWebSocket::Connect" />
   /// </summary>
-  void Connect(const Protocols &protocols, const Options &options) override;
+  void Connect(const Protocols& protocols, const Options& options) override;
```

Comment:
Why are we exempting the file from clang-format and changing the formatting? If we want to do something else, could we add a new clang-format file in a separate change instead of combining changes into this?

Reply:
Because these particular boost-using files get messed up by clang-format.
Keeping these exceptions makes them easier to read and maintain.